tests/lora/test_lora_layers_sd3.py

```diff
@@ -175,36 +175,36 @@ def test_sd3_img2img_lora(self):
         image_slice = image[0, :10, :10]
         expected_slice = np.array(
```

Comment:
Can we follow this format for the output slices?
https://github.com/huggingface/diffusers/blob/13e8fdecda91e27e40b15fa8a8f456ade773e6eb/tests/lora/test_lora_layers_flux.py#L236

Reply:
fixed format